experimental/host/beagle/black/black.go

```diff
@@ -5,12 +5,14 @@
 package black
 
 import (
+	"bytes"
 	"errors"
-	"os"
+	"io/ioutil"
```

Comment:
I recommend to integrate goimports to run on save in your text editor so imports are updated automatically.